optimizely/config_manager.py

```diff
@@ -187,10 +194,12 @@ def __init__(self,
         self.datafile_url = self.get_datafile_url(sdk_key, url,
                                                   url_template or enums.ConfigManager.DATAFILE_URL_TEMPLATE)
         self.set_update_interval(update_interval)
+        self.set_blocking_timeout(blocking_timeout)
         self.last_modified = None
         self._polling_thread = threading.Thread(target=self._run)
         self._polling_thread.setDaemon(True)
         self._polling_thread.start()
+        self._condition = threading.Condition()
```

Comment:
You will have to put this inside static config manager since you use it in it's method.